### PR TITLE
Add new fields in user form for further infos

### PR DESCRIPTION
**Pull request by CFarcy**

### CODE REVIEW

File: src/modules/RA/User/components/UserFields.js [hunk 1]
```diff
@@ -1,35 +1,78 @@
...
+  return (
+    <SimpleForm {...props}>
+      {edit && <TextInput disabled source="id" />}
+      {edit && <TextInput disabled source="uuid" />}
+
+      <TextInput source="properties.first_name" label="user.form.firstname" />
+      <TextInput source="properties.last_name" label="user.form.lastname" />
+      <TextInput source="email" type="email" />
+      {!edit && <TextInput source="password" type="password" />}
+      <BooleanInput source="is_superuser" />
```
Comment (dtrucs) [inline src/modules/RA/User/components/UserFields.js:39]: Maybe it should have a condition to display/control this boolean? Such as only superUser can access/edit this setting

Comment (CFarcy) [reply src/modules/RA/User/components/UserFields.js:39]: Real question: User creation shouldn't it be accessible to superUser only?

Comment (jrmi) [reply src/modules/RA/User/components/UserFields.js:39]: User creation should be accessible to user with "user_management" permissions only. That the point of permissions field that list functional permissions that imply available modules. 
Forbidden modules shouldn't be displayed to user. For now, if a user can manage users, he can create super users. May be later we'll make something to protect that.

**Review by jrmi — APPROVED** [review]

**Review by jrmi — DISMISSED** [review]

File: src/modules/RA/User/components/UserFields.js [hunk 2]
```diff
@@ -1,35 +1,79 @@
...
+  return (
+    <SimpleForm {...props}>
+      {edit && <TextInput disabled source="id" />}
+      {edit && <TextInput disabled source="uuid" />}
+
+      <TextInput source="properties.first_name" label="user.form.firstname" />
+      <TextInput source="properties.last_name" label="user.form.lastname" />
+      <TextInput source="email" type="email" />
+      {!edit && <TextInput source="password" type="password" />}
+      { isSuperUser && <BooleanInput source="is_superuser" />}
```
Comment (dtrucs) [inline src/modules/RA/User/components/UserFields.js:40]: It looks good to me!
Maybe it's better to display the field but disabled in order to escort the user. Something like this ⬇️ 
```jsx
<BooleanInput 
  source="is_superuser" 
  disabled={!isSuperUser}
  helperText={!isSuperUser && 'You don't have enough permission to edit this field' }
/>
```
 

```

File: src/modules/RA/User/components/UserFields.js (hunk 1 above)
Comment (jrmi) [reply src/modules/RA/User/components/UserFields.js]: We discuss this IRL !

**Review by dtrucs — APPROVED** [review]

File: src/modules/RA/User/components/UserFields.js (hunk 2 above)
Comment (CFarcy) [reply src/modules/RA/User/components/UserFields.js]: Done!

**Review by dtrucs — APPROVED** [review]

**Pull request merged**